src/app/modules/item/services/item-task-answer.service.ts

```diff
@@ -62,13 +67,20 @@ export class ItemTaskAnswerService implements OnDestroy {
     distinctUntilChanged((a, b) => a.answer === b.answer && a.state === b.state),
     skip(1), // avoid saving an answer right after fetching it
     withLatestFrom(this.config$),
-    switchMap(([{ answer, state }, { route, attemptId }]) => this.currentAnswerService.update(route.id, attemptId, { answer, state })),
+    repeatLatestWhen(fromEvent(globalThis, 'online')),
+    switchMap(([{ answer, state }, { route, attemptId }]) => this.currentAnswerService.update(route.id, attemptId, { answer, state }).pipe(
+      retryAfter({ delay: answerAndStateSaveRetryDelay, maxAttempts: answerAndStateSaveRetries }),
```

Comment:
After these `maxAttempts: answerAndStateSaveRetries`, what does it do? It stops retrying? Actually, as it is a low-frequence save, it may continue as long as it is not saved.